<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBoxEcuId">
     <property name="text">
      <string>ECU Id:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="lineEditEcuId"/>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="checkBoxApplicationId">
     <property name="text">
      <string>Application Id:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="lineEditApplicationId">
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="checkBoxHeaderText">
     <property name="text">
      <string>Header Text:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLineEdit" name="lineEditHeaderText"/>
   </item>
   <item row="12" column="0">
    <widget class="QCheckBox" name="checkBoxLogLevelMax">
     <property name="text">
      <string>Log Level Max</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QComboBox" name="comboBoxLogLevelMax">
     <item>
      <property name="text">
       <string>off</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fatal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>error</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>warn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>info</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>debug</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>verbose</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Marker colour:</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QComboBox" name="comboBoxFilterColour">
     <item>
      <property name="text">
       <string>green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>red</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>yellow</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>blue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>light grey</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dark grey</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBoxContextId">
     <property name="text">
      <string>Context Id:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEditContextId">
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="checkBoxPayloadText">
     <property name="text">
      <string>Payload Text:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="lineEditPayloadText"/>
   </item>
   <item row="12" column="1">
    <widget class="QCheckBox" name="checkBoxLogLevelMin">
     <property name="text">
      <string>Log Level Min</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QComboBox" name="comboBoxLogLevelMin">
     <item>
      <property name="text">
       <string>off</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fatal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>error</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>warn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>info</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>debug</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>verbose</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="comboBoxType">
     <item>
      <property name="text">
       <string>positive</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>negative</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>marker</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QCheckBox" name="checkBoxCtrlMsgs">
     <property name="text">
      <string>Control Messages</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
